src/server/network/socket-gameplay.js

```diff
@@ -0,0 +1,55 @@
+const socketsHelper = require("./sockets-helper");
+const log = require("../../log");
+const db = require("../database/manager");
+
+const socketGameplay = function(getInitialDataPayload) {
+	return new socketsHelper.Socket(
+		// Route
+		"/gameplay",
+
+		// Options
+		{
+			compression: 0,
```

Comment:
There seems to be no issues if we enable compression, so I'd say it worth to turn it on?

Reply:
Since we already compress it on our end, this will only slow things down. Unless you observed some benefits?

Reply:
We currently don't compress gameplay data with `pako`, we just pack it together with MessagePack and leave it at that! It could do with further testing, but I'm making the assumption that `pako` may not be optimised for performance as much as network-specific compression.

Reply:
Ah, I thought we did! Then it might be a good thing to look into.